german/Singen/Welle/welle.md

```diff
@@ -0,0 +1,5 @@
+Sie trinken viel Salzwasser und aufhören zu singen.
+Sie störte die Stetigkeit des Wassers, das die Aufmerksamkeit eines Raubfische bekommt.
+Dieser Fisch wurde von einer Gruppe von Walen gejagt.
+
+[Der Fisch entkommt und lässt Sie zu den Walen.](Walen/walen.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.